pkg/component/component.go

```diff
@@ -372,17 +421,36 @@ func PushLocal(client *occlient.Client, componentName string, applicationName st
 	}
 	targetPath := fmt.Sprintf("%s/src", s2iSrcPath)
 
+	// If there are files identified as deleted, propogate them to the component pod
+	if len(delFiles) > 0 {
+		s := log.Spinner("deleting files from component")
```

Comment:
I don't think that this is a message that user should see. 
It is expected that when you delete something it will be deleted ;-)

Reply:
Ack